### PR TITLE
[SDPA-4491][SDPA-4523] Added Open form component to landing page

### PR DESCRIPTION
**Pull request by tim-yao**
**JIRA issue:** https://digital-engagement.atlassian.net/browse/SDPA-4491
https://digital-engagement.atlassian.net/browse/SDPA-4523

### Changed

1.  Added Open form component to Landing page
2. Added snapshot tests for OpenForm

### CODE REVIEW

**Comment by tim-yao** [comment]
I will add snapshot test for example form later on this PR, please CR first so I can move to QA.

File: packages/ripple-nuxt-tide/modules/landing-page/components/TideOpenForm.vue [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    document.head.appendChild(openFormsEmbedScript)
+  }
+}
+</script>
+
+<style lang="scss">
+@import "~@dpc-sdp/ripple-global/scss/settings";
+
+.tide-open-form {
+  // This OpenForm component will cause high CLS(cumulative layout shift) issue.
```
Comment (rubyllamadora) [inline packages/ripple-nuxt-tide/modules/landing-page/components/TideOpenForm.vue:25]: Nice, how to know this thing? I should consider this as well when updating/building new feature.

**Review by rubyllamadora — APPROVED** [review]
Looks straightforward to me @tim-yao

File: packages/ripple-nuxt-tide/modules/landing-page/components/TideOpenForm.vue (hunk 1 above)
Comment (tim-yao) [reply packages/ripple-nuxt-tide/modules/landing-page/components/TideOpenForm.vue:25]: @rubyllamadora Only happens when a client JS changes the layout and we should avoid it if possible. As this is in OpenForms so I can't do too much. You can install https://chrome.google.com/webstore/detail/web-vitals/ahfhijdlegdabablpippeagghigmibma?hl=en to measure it.

Currently the alerts on page top cause high CLS issue on mobile which we need to fix it as well.

**Comment by tim-yao** [comment]
Hey @kavyajain-qa I added a image snapshot test for openform here, it will be running nightly.

**Comment by dylankelly** [comment]
> Hey @kavyajain-qa I added an image snapshot test for openform here, it will be running nightly.

@tim-yao This isnt a good long term solution as we have to add image snapshots into git, please dont overuse this. We have looked at services like percy.io for this which is what we should use when we can get some money.

**Comment by tim-yao** [comment]
> > Hey @kavyajain-qa I added an image snapshot test for openform here, it will be running nightly.
> 
> @tim-yao This isnt a good long term solution as we have to add image snapshots into git, please dont overuse this. We have looked at services like percy.io for this which is what we should use when we can get some money.

Good point, this will be a simple solution before we get any budget for paid services. Usage should be very minor at the moment, only few important pages to help QA.

**Pull request merged**